internal/service/project/resource_project.go

```diff
@@ -603,23 +603,23 @@ type AdditionalProperties struct {
 }
 
 // GetProjectPropsFromAPI fetches properties obtained from complementary endpoints associated with a project.
-func GetProjectPropsFromAPI(ctx context.Context, client GroupProjectService, projectID string) (*AdditionalProperties, error) {
-	teams, _, err := client.ListProjectTeams(ctx, projectID)
+func GetProjectPropsFromAPI(ctx context.Context, client *admin.APIClient, projectID string) (*AdditionalProperties, error) {
```

Comment:
in the PR @AgustinBettati did, he passed the API itself, TeamsApi in this case instead of client, WDYT?